### ISSUE TITLE
Can't receive Chat And Game invite

### ISSUE THREAD

**Issue by nongnoobjung** [fjxokt/lolClient#5]
Can't receive Chat And Game invite

**Comment by emabrey** [comment]
It seems that this project has died. My pull request has been pending for months and there haven't been additional contributions. I am working on a separate project currently that will allow me to create an open source league client. But for now, I would say not to expect this project to advance. I'm not terribly fond of the codebase either, so my project is looking to be a complete rewrite. Thank you for opening the issue however. I am not going to close the issue seeing as to how, theoretically, this project could still be updated.

**Comment by kpyfan** [comment]
Hey, if you happen to need any help, I'm definitely still willing to help
out.

**Comment by emabrey** [comment]
Yes, I would very much like some help. Currently I am writing a Java library for a lightweight RTMP client, which will need to be finished before I can work on the actual LoL client. Send me an email at emilymabrey93(at)gmail.com . If you want to help with the RTMP library feel free to submit pull requests and open issues on that project as well.

**Comment by eddy5641** [comment]
@kpyfan 
If you want to help me out on a different project, for league of legends, https://github.com/eddy5641/LegendaryClient/ is snowl's code but more up to date. I am unable to help with java because I never learned it but I know it is super similar to c#, the code that LegendaryClient is coded in
